wordpress-mobile/WordPress-Aztec-iOS
# Removing NSLinkAttribute when entering text after or before the anchor

### Details:
In this PR we're implementing a simple mechanism that will remove the `NSLinkAttribute` whenever we're not editing text **within** a Link's Anchor. 

Meaning that: inserting a link, and entering some text immediately after it will no longer further extend the linkified range.

Closes #251

### Testing:
1. Launch Aztec
2. Insert a link
3. Add some text. Verify it does not get linkified
4. Move the cursor at the beginning of the link
5. Enter `Some Random Text` and verify it does not get linkified
6. Verify that any newly entered text within the Link's Anchor gets properlylinkified

Needs Review: @SergioEstevao 
Thanks in advance sir!
